lib/CampMinder/version.rb

```diff
@@ -1,3 +1,3 @@
 module CampMinder
-  VERSION = "0.0.1"
+  VERSION = '0.0.2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.